package.json

```diff
@@ -37,25 +37,25 @@
     "babel-core": "^6.4.5",
     "babel-preset-es2015": "^6.3.13",
     "bluebird": "^3.1.1",
-    "gulp": "^3.9.0",
+    "gulp": "^4.0.2",
     "gulp-babel": "^6.1.2",
     "gulp-changed": "^1.3.0",
-    "gulp-eslint": "^1.1.1",
-    "gulp-xo": "^0.7.0",
+    "gulp-eslint": "^6.0.0",
+    "gulp-xo": "^0.21.0",
     "load-plugins": "^2.1.0",
-    "mocha": "^2.2.5",
+    "mocha": "^6.1.4",
     "moment": "^2.10.3",
     "request": "^2.58.0",
     "should": "^6.0.3",
     "vorpal-less": "0.0.4",
-    "vorpal-repl": "^1.1.8",
-    "xo": "^0.9.0"
+    "vorpal-repl": "github:joshmanders/vorpal-repl#fix-vulnerabilities",
```

Comment:
Depending on how this fork is published, we can do the same if the `moleculerjs` org forks `vorpal-repl` for the org instead of my personal account.

Reply:
Opening up a PR to Vorpal itself is also a good idea. I do not believe we should be using references to personal forks.

Reply:
Agreed, but the issue right now is that nobody who can merge the PR's in for vorpal are active, so the project is dead in the water. Only solution is to publish from branches like this, or publish our own packages to utilize.

Reply:
I'm forking vorpal-repl, as well...

Reply:
Here you go: https://github.com/moleculerjs/vorpal-repl

Reply:
By the way, the source of vorpal-repl is only [44 lines](https://github.com/moleculerjs/vorpal-repl/blob/master/lib/repl.js). Would be better to move this code into vorpal directly, and minus 1 dependency :)

Reply:
I can move it into vorpal for my PR if you'd like, or I can PR into the repl directory. Whatever you want is fine

Reply:
Great! I prefer moving the vorpal-repl into vorpal. Too small code for a separated repo :)

Reply:
Moved it into this PR.